AuxTel/AuxTel-Standard-Operations/AuxTel-Daytime-Operations/Daytime-Checkout.rst

```diff
@@ -4,10 +4,16 @@
 
 .. _AuxTel-DayTime-Operations-Daytime-Checkout:
 
+.. Links 
+
+.. _`summit's RubinTV`: https://summit-lsp.lsst.codes/rubintv/summit/auxtel 
+.. _`USDF RubinTV`: https://usdf-rsp-dev.slac.stanford.edu/rubintv 
```

Comment:
Although the link works is not the right one. At June 3 USDF data ingestion/RubinTV of AuxTel data is located in https://usdf-rsp-dev.slac.stanford.edu/rubintv/summit-usdf/auxtel

Reply:
Many thanks for the careful review!